lib/api_browser/app/js/filters/attribute_name.js

```diff
@@ -1,8 +1,9 @@
 app.filter('attributeName', function($sce) {
   return function(input) {
     var parts = input.split('.');
-    if (parts.length == 2) {
-      return $sce.trustAsHtml('<span class="attribute-prefix">' + parts[0] + '.</span>' + parts[1]);
+    if (parts.length > 1) {
+      var prefix = parts.slice(0,parts.length-1).join('.')
```

Comment:
ya, that doesn't quite work, but I've fixed it appropriately. Hopefully that's idiomatic enough.

As far as not showing the parents...I think that's something we should wait on, and discuss further, to see what options we have...I'm not sure that not showing the paren attributes will be a good thing always. I'd wait until that problem actually exists, and wait to try a couple ways to visualize it whenever that comes about.